mockgen/internal/tests/defined_import_local_name/input.go

```diff
@@ -0,0 +1,13 @@
+package defined_import_local_name
+
+import (
+	"bytes"
+	"context"
+)
+
+//go:generate mockgen -package defined_import_local_name -destination mock.go -source input.go -imports b_mock=bytes,c_mock=context
+
+type WithDotImports interface {
```

Comment:
Could we add an additional package to test for stdlib and external imports?

Suggestion: Update the method names to reflect which scenario being tested.